paasta_tools/cli/cmds/logs.py

```diff
@@ -1333,6 +1333,9 @@ def paasta_logs(args: argparse.Namespace) -> int:
         instances = None
     else:
         instances = args.instances.split(",")
+        # Check for flint instance names only, not the containers inside them
+        check_flink_instances = [x.split(".")[0] for x in instances if x.split(".")]
```

Comment:
A couple thoughts:
* Although we know that these are probably flink instances, the log suffix behavior is something that other instance types could use, too.  I think it'd be better to call this something like `instances_without_suffixes` (which I know doesn't really roll off the tongue), reference PAASTA-15311 in the comment, and only talk about flink as an example use case, if at all
* if there is no period character, `split()` will return a 1-element list, so you don't actually need the `if` part of this -- you can just access `x.split(".")[0]`